packages/pn-personafisica-webapp/src/components/Contacts/SpecialContacts.tsx

```diff
@@ -359,146 +257,65 @@ const SpecialContacts: React.FC<Props> = ({ digitalAddresses }) => {
       reloadAction={fetchAllActivatedParties}
       mainText={t('special-contacts.fetch-party-error', { ns: 'recapiti' })}
     >
-      <DigitalContactsCard title="" subtitle={t('special-contacts.subtitle', { ns: 'recapiti' })}>
-        <Typography sx={{ marginTop: '20px' }}>{t('required-fields')}</Typography>
-        <form
-          style={{ margin: '20px 0' }}
-          onSubmit={formik.handleSubmit}
-          data-testid="specialContact"
-        >
-          <Stack direction={{ xs: 'column', lg: 'row' }} spacing={2}>
-            <PnAutocomplete
-              id="sender"
-              data-testid="sender"
-              size="small"
-              options={parties ?? []}
-              autoComplete
-              getOptionLabel={getOptionLabel}
-              noOptionsText={t('common.enti-not-found', { ns: 'recapiti' })}
-              isOptionEqualToValue={(option, value) => option.id === value.id}
-              onChange={senderChangeHandler}
-              inputValue={senderInputValue}
-              onInputChange={(_event, newInputValue, reason) => {
-                if (reason === 'input') {
-                  setSenderInputValue(newInputValue);
-                }
+      {digitalAddresses &&
```

Comment:
move the ApiErrorWrapper inside the addSpecialContactDialog and try to wrap the autocomplete field. check how it looks